doc/chapters/gateways/10-cgroups-gateway.rst

```diff
@@ -38,6 +38,13 @@ memory, thus it cannot be less than ``memory.limit_in_bytes``.
 Also note that the one can use a suffix (k, K, m, M, g or G) to indicate values in kilo,
 mega or gigabytes when setting ``memory.limit_in_bytes`` or ``memory.memsw.limit_in_bytes``.
 
+Setting network classes
+-----------------------
+If you want to mark packets from containers, you can set the ``net_cls.classid`` to a hexadecimal
+value of the form 0xAAAABBBB (leading zeroes may be omitted for both A and B part). The gateway
+supports and checks the syntax for this key. More information about this can be found `in the Linux
+kernel docs <https://www.kernel.org/doc/Documentation/cgroup-v1/net_cls.txt>`_.
+
```

Comment:
I think this should be extended with an example. The example config section doesn't reflect this at all.